clients/python-apiserver-client/README.md

```diff
@@ -0,0 +1,31 @@
+# Python client for KubeRay API server
+
+This Python client is currently only supporting Ray cluster management through usage of the `API server` Ray API. It implements all of the current functionality of the API server (and the operator) and provide pythonic APIs to the capabilities.
+
+The package supports well documented in the code Python objects that can be used to build and receive payloads for creation, listing and deleting of [template](https://ray-project.github.io/kuberay/components/apiserver/#compute-template) and Ray clusters.
+
+The main class of the package is [KubeRayAPIs](python_apiserver_client/kuberay_apis.py) that implements all of the functionality. It leverages [templates](python_apiserver_client/params/templates.py) and [cluster](python_apiserver_client/params/cluster.py) definitions, allowing to specify all required parameters as straight Python classes. Additional (intermediate) definitons are provided (see [environment variables](python_apiserver_client/params/environmentvariables.py), [volumes](python_apiserver_client/params/volumes.py), [head group](python_apiserver_client/params/headnode.py) and [worker group](python_apiserver_client/params/workernode.py))
+
+## Prerequisites
+
+It is also expected that the `kuberay operator` ([Installation instructions are here.](https://github.com/ray-project/kuberay#quick-start)) and `api server` ([Installation instructions are here.](https://ray-project.github.io/kuberay/components/apiserver)) are installed
+
+## Development
+
+Start from installing `setup-tools`
```

Comment:
"Start by ..."

Reply:
Also, is `setup-tools`/`setuptools` a very common name that might be confused with other "setup tools"? Would it make sense to add a unique prefix to the name or some other change?

Reply:
It has a specific pip install